ddtrace/contrib/asyncio/compat.py

```diff
@@ -0,0 +1,8 @@
+try:
+    from asyncio import current_task as asyncio_current_task
```

Comment:
Is there a version number we can use to determine where these should be rather than relying on `ImportError`?

This is likely fine, but might be cleaner as:

```python
if asyncio.VERSION > (1, 0, 0):
    from asyncio import current_task as asyncio_current_task

...
```

Reply:
What you have here is likely fine, this is just a mild suggestion.